AppDB/zkappscale/zktransaction.py

```diff
@@ -118,7 +118,7 @@ class ZKTransaction:
   """
 
   # The number of times we should retry ZooKeeper operations, by default.
-  DEFAULT_NUM_RETRIES = 0
+  DEFAULT_NUM_RETRIES = 5
```

Comment:
You were right, I think it was just waiting on the ZooKeeper to start. Maybe it was too quick before and I am now sleeping a bit to make sure ZK and Cassandra are up before making the call to ZKTransaction. Will remove this!